.github/workflows/ci.yml

```
@@ -31,55 +31,24 @@ jobs:
       - name: Run YAML Lint
         uses: actionshub/yamllint@main
 
-  build:
-    name: Build
-    runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        otp_version: ['27.1']
-
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: erlef/setup-beam@v1
-        with:
-          otp-version: ${{ matrix.otp_version }}
-          version-type: 'strict'
-
-      - uses: actions/cache@v4
-        with:
-          path: |
-            ~/.cache/rebar3
-            _build
-          key: ${{ runner.os }}-erlang-${{ matrix.otp_version }}-${{ hashFiles('**/*rebar.lock') }}
-
-      - run: make build
-
   test:
-    name: Test
-    runs-on: ubuntu-22.04
+    name: OTP ${{matrix.otp}}
     strategy:
       matrix:
-        otp_version: ['27.1']
-
-    needs:
-      - build
-      - markdownlint-cli
-      - yamllint
-
+        otp: ['27']
+        rebar3: ['3.24.0']
```

We can even relax it to the mini version, if supported.

```suggestion
        rebar3: ['3.24']
```

---

Good point, I believe it is supported, let me push that 👌🏽